### PR TITLE
Fix error UI pane string override

### PR DESCRIPTION
**Pull request by kevinlin98z**

### CODE REVIEW

File: chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx [hunk 1]
```diff
@@ -33,13 +33,13 @@ export const LoadingPaneStateful = (props: ILoadingPaneProps) => {
     };
 
     const errorUIControlProps: ILoadingPaneControlProps = {
+        ...props.controlProps,
         id: "oc-lcw-alert-pane",
         dir: state.domainStates.globalDir,
-        titleText: "Chat is failing to load.",
-        subtitleText: "Please Close the chat and try again.",
+        titleText: "We are unable to load chat at this time.",
```
Comment (charliewang95) [inline chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx:39]: "load chat" or "load the chat"

Comment (kevinlin98z) [reply chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx:39]: String has been confirmed to be the above by PM.

**Review by abhinab22mohanty — APPROVED** [review]

File: chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx (hunk 1 above)
Comment (akagar-microsoft) [inline chat-widget/src/components/loadingpanestateful/LoadingPaneStateful.tsx:39]: Are we suppose to hard-code the strings ? I mean how does localization works in this case ?

**Comment by kevinlin98z** [comment]
PR will soon be discarded.

**Pull request closed without merging**